code/datums/components/barber.dm

```diff
@@ -87,22 +93,29 @@
 			return 0
 
 	SPAWN_DBG(0)
-		var/list/region = list("First Hairea" = 1, "Second Hairea" = 2, "Third Hairea" = 3)
+		var/list/region = list(
+			"Bottom Hairea ([M.bioHolder.mobAppearance.customization_first.name])" = BOTTOM_DETAIL,
+			"Middle Hairea ([M.bioHolder.mobAppearance.customization_second.name])" = MIDDLE_DETAIL,
+			"Top Hairea ([M.bioHolder.mobAppearance.customization_third.name])" = TOP_DETAIL,
```

Comment:
not a fan of this naming, it will be unclear to new players what this means

Reply:
I think it's better than what it was before, which was just numbered, this is at least consistent with what they select in character setup.

What do you think would be better?

Reply:
Oh, you just mean the "Hairea" pun? Sure, we can change this to be "X Detail", like in Character Setup.

Reply:
![image](https://user-images.githubusercontent.com/35579460/121126379-5d083c80-c828-11eb-9415-c6906348e45b.png)
![image](https://user-images.githubusercontent.com/35579460/121126430-6d201c00-c828-11eb-9cbb-07b59cc79cbc.png)
Like so?